config/vufind/config.ini

```diff
@@ -840,6 +847,14 @@ authors         = Wikipedia
 url = "https://api.vlb.de/api/v1/cover/"
 ; token = "XXXXXX-XXXX-XXXXX-XXXXXXXXXXXX"
 
+; This section is needed for LocalFile cover loading.
+; Note:  All coverimage LocalFile:ConfigSection must include the following keys: 
+;        filepath, filename, and imagetype.
+[LocalFile]
+filepath = /usr/local/vufind/local/coverimages
+filename = recordid
+imagetype = jpg
```

Comment:
Is there a strong reason to include imagetype as a separate setting rather than including it in the filename?